cmd/node/main.go

```diff
@@ -642,7 +643,7 @@ func startNode(ctx *cli.Context, log *logger.Logger, version string) error {
 		return err
 	}
 
-	ef := facade.NewElrondNodeFacade(currentNode, apiResolver)
+	ef := facade.NewElrondNodeFacade(currentNode, apiResolver, !useTermui)
```

Comment:
For readability, you could write above: `ginDebugMode := !useTermui` then pass it as the argument to `NewElrondNodeFacade`.

Reply:
ok

Reply:
done